sdk/storage/ustorageclient/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 3.2)
+
+project(azure_iot_ustorageclient)
+
+set(AZURE_IOT_USTORAGECLIENT_VERSION 1.0.1)
+
+#Include helper functions
+include("build_all/configs/ustorageclient_functions.cmake")
+include("build_all/configs/ustorageclient_build_rules.cmake")
+
+#Set the standards for C and C++
+add_standards()
+
+#Specify user options
+option(run_azstorage_e2e_tests "set run_e2e_tests to ON to run e2e tests (default is OFF)" OFF)
```

Comment:
I prefer to just use BUILD_TESTING (which is a standard cmake thing) but it's not a big deal at all